### PR TITLE
Update freie-universitat-berlin-geographische-wissenschaften.csl

### PR DESCRIPTION
**Pull request by Sidapo**
now follows the documentation more closely + some code clean up

### CODE REVIEW

File: freie-universitat-berlin-geographische-wissenschaften.csl [hunk 1]
```diff
@@ -17,35 +17,38 @@
...
     </choose>
-    <text variable="publisher-place"/>
   </macro>
   <macro name="pagenumbers">
-    <group delimiter=" ">
-      <number variable="number-of-pages"/>
-      <label variable="number-of-pages" form="short"/>
-    </group>
+    <number variable="number-of-pages" suffix=" "/>
+    <label variable="number-of-pages" form="short"/>
```
Comment (zuphilip) [inline freie-universitat-berlin-geographische-wissenschaften.csl]: That doesn't look like an improvement. Why did you delete the group? Now it will display the label even the data for the number of pages is not present. There are more such cases following...

**Comment by Sidapo** [comment]
oops, I didn't realize that would happen. I was under the impression my style had a lot of unnecessary groups... I'll undo the group changes then

File: freie-universitat-berlin-geographische-wissenschaften.csl [hunk 2]
```diff
@@ -17,35 +17,40 @@
...
-    </group>
+     </group>
   </macro>
   <macro name="thesis">
-    <group prefix="Dissertation, " delimiter=". " suffix=".">
-      <text variable="publisher"/>
-      <text macro="pagenumbers"/>
-    </group>
+    <text variable="publisher" prefix="Dissertation, " suffix=". " />
+    <text macro="pagenumbers" suffix="."/>
```
Comment (zuphilip) [inline freie-universitat-berlin-geographische-wissenschaften.csl]: Another destroyed group

File: freie-universitat-berlin-geographische-wissenschaften.csl [hunk 3]
```diff
@@ -113,8 +123,8 @@
   </macro>
   <macro name="journal">
     <group delimiter=". " suffix=".">
-      <group delimiter=" ">
-        <text variable="container-title"/>
+      <group delimiter=" "> 
+        <text variable="container-title" suffix=" "/>
         <number variable="volume" font-weight="bold"/>
       </group>
```
Comment (zuphilip) [inline freie-universitat-berlin-geographische-wissenschaften.csl]: The suffix is not needed here, because you already have it as a delimiter.

File: freie-universitat-berlin-geographische-wissenschaften.csl (hunk 2 above)
Comment (Sidapo) [inline freie-universitat-berlin-geographische-wissenschaften.csl]: ..but isn't this group actually unnecessary? Since the page numbers should still be displayed even if the publisher is not written.

File: freie-universitat-berlin-geographische-wissenschaften.csl [hunk 4]
```diff
@@ -180,10 +188,8 @@
       <key macro="author"/>
     </sort>
     <layout prefix="(" suffix=")" delimiter="; ">
-      <group delimiter=" ">
-        <text macro="author-short"/>
-        <text macro="year-date"/>
-      </group>
+      <text macro="author-short" suffix=" "/>
+      <text macro="year-date"/>
```
Comment (zuphilip) [inline freie-universitat-berlin-geographische-wissenschaften.csl]: Another destroyed group

File: freie-universitat-berlin-geographische-wissenschaften.csl (hunk 2 above)
Comment (zuphilip) [inline freie-universitat-berlin-geographische-wissenschaften.csl]: Right, but the suffixes are different. There are tree cases possible i) both variables are there, ii) only publisher is there, iii) only pagenumbers is there. The delimiter in the group will only be shown if both occur.

File: freie-universitat-berlin-geographische-wissenschaften.csl (hunk 4 above)
Comment (Sidapo) [inline freie-universitat-berlin-geographische-wissenschaften.csl]: here I thought the same as above. Since "author-short" always returns something, the group is superfluous..

File: freie-universitat-berlin-geographische-wissenschaften.csl (hunk 2 above)
Comment (zuphilip) [inline freie-universitat-berlin-geographische-wissenschaften.csl]: In iii) the pagenumbers will still be shown, but the prefix "Dissertation, " is missing.

**Comment by zuphilip** [comment]
Here is a really good blog entry explaining groups in CSL: https://zoteromusings.wordpress.com/2013/10/28/writing-csl-features-and-best-practices/

File: freie-universitat-berlin-geographische-wissenschaften.csl (hunk 4 above)
Comment (zuphilip) [inline freie-universitat-berlin-geographische-wissenschaften.csl]: Assume "year-date" is missing, then we will still have the space from the suffix in your current code.

**Comment by Sidapo** [comment]
okay great, thanks for your help and patience! I'll change the groups back then.. Classic example of "don't fix whats not broken" I guess

**Comment by zuphilip** [comment]
Okay, you fixed it, thank you. Let us wait for one of the maintainer to review here.

**Comment by adam3smith** [comment]
looks all good to me, thank you both!
